src/app/portfolio/_components/ProjectCard.tsx

```
@@ -59,6 +84,35 @@ export default function ProjectCard({
+        <div className="flex gap-4">
+          <button
+            onClick={() => {
+              DeleteProjectMutation.mutateAsync(id);
+              modalId && closeModal(modalId);
+            }}
+            className="grow rounded bg-brand-primary px-4 py-2 text-text-primary hover:bg-brand-secondary dark:hover:bg-brand_dark-secondary"
+          >
+            삭제
+          </button>
+          <button
+            onClick={() => modalId && closeModal(modalId)}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)